schemas/host-config.json

```diff
@@ -360,6 +360,16 @@
 				},
 				"errorMessage": {
 					"$ref": "#/definitions/ErrorMessageConfig"
+				},
+				"validationBehavior":{
```

Comment:
How about calling it `validationBegins` or `initialValidation`?

At first glance, the name didn't give me the impression that subsequent validation wouldn't be affected by this.

Reply:
Good point. I like `initialValidation`.

---
In reply to: [472678974](https://github.com/microsoft/AdaptiveCards/pull/4603#discussion_r472678974) [](ancestors = 472678974)